### ISSUE TITLE
How to switch off connected tooltips in case of connected charts?

### ISSUE THREAD

**Issue by rdatasculptor** [JohnCoene/echarts4r#264]
I really like the combination between the features `e_connect_group()` and `timeline = TRUE`. If the groups of the timeline are the same for the connected charts, you only need to show just one timeline in order to navigate between groups. The connection makes sure alle the connected charts will show content based on the selected group, in one go.

However, there seems a side effect. When using in e.g. two the connected charts the `e_tooltip()` feature hovering over chart 1 will show the tooltips in chart2 as well. In my case this doesn't make mutch sense. The content of the tooltips are not always connected.

This is an example of what I mean:

```
df1 <- data.frame(x=1:10, y=c(2,3,4,1,2,7,6,3,4,5), group=c(1,1,2,2,1,1,2,2,1,1))
df2 <- data.frame(a=1:5,b=c(1,2,1,4,7), group=c(1,1,1,2,2))

library(echarts4r)
library(manipulateWidget)

chart1 <- df1 %>%
  group_by(group) %>%
  e_chart(x, timeline = TRUE) %>%
  e_timeline_opts(show = FALSE) %>%
  e_tooltip() %>%
  e_group("chartgroup") %>%
  e_bar(y) %>% 
  e_connect_group("chartgroup")

chart2 <- df2 %>%
  group_by(group) %>%
  e_chart(a, timeline = TRUE) %>%
  e_tooltip() %>%
  e_group("chartgroup") %>%
  e_line(b) %>% 
  e_connect_group("chartgroup")

combineWidgets(chart1, chart2)

```

Hovering over the bars in chart 1 shows both tooltips in chart 1 and chart2. But the variable b is just another variable than y. So the connections between the tooltips doesn't make sense here.

Is there a way to switch off the connection between the tooltips in case of connected charts?

Many thanks in advance!

**Comment by JohnCoene** [comment]
Yes indeed, that connection is nonsensical but I'm afraid I see no way to turn this on or off in the [official  documentation](https://echarts.apache.org/en/api.html#echarts.connect)

**Comment by helgasoft** [comment]
The two charts are not related in any logical way, so why have them connected?
Maybe this is what you are after ? :
```r
df1 <- data.frame(x=1:10, y=c(2,3,4,1,2,7,6,3,4,5), group=c(1,1,2,2,1,1,2,2,1,1))
df2 <- data.frame(a=1:5,b=c(1,2,1,4,7), group=c(1,1,1,2,2))

library(echarts4r)
library(manipulateWidget)

chart1 <- df1 %>%
  group_by(group) %>%
  e_chart(x, timeline = TRUE) %>%
  e_timeline_opts(show = FALSE) %>%
  e_tooltip() %>%
  e_group("chartgroup") %>%
  e_bar(y)

chart2 <- df2 %>%
  group_by(group) %>%
  e_chart(a, timeline = TRUE) %>%
  e_tooltip() %>%
  e_group("chartgroup") %>%
  e_line(b)

combineWidgets(chart1, chart2)
```
PS: I didn't know library _combineWidgets_ - cool one 👍

**Comment by rdatasculptor** [comment]
Thank you for your response!

the charts are connected based on their group membership. I mean the groups
of the timeline. This example is ofcourse meaningless, but in "real" live I
use this a lot in my reports. Sometimes there are groups that can be
defined by scores on unrelated variables. This way of connecting charts
makes it possible to show the scores on different variables per group in
one go.
Unfortenately the tooltip makes it sometimes a bit messy, like the ones in
my example.

Yes, I agree that the combinWidgets is very cool! I use it when I want to
place more than one e_chart in one cell of a flexcashboard.. It works very
neatly.


Op di 19 jan. 2021 om 19:12 schreef helgasoft <notifications@github.com>:
